### DISCUSSION TITLE
[Projects Beta] Filtering issues by a certain value OR no value

### DISCUSSION THREAD

**Discussion by brettwillis** [community/community#16941] [Projects and Issues]
Say I want to filter a view to see all issues that have either a certain value (e.g. `phase:"Phase 1"`), OR have no value (e.g. `no:phase`). Currently it is not possible, because `phase:"Phase 1",no:phase` is not a valid OR combination.

Propose a syntax something like `phase:@none` as a valid alternative to `no:phase` which would allow an OR combination like `phase:@none,"Phase 1"`.

**Comment by tdtm** [comment]
Same idea for me, but my particular use-case would be for `assignee`. 

More specifically, `assignee:@me` with `assignee:@none` so I can see everything that's already assigned to me + things that nobody's assigned and that I may pick up.

**Comment by shreve** [comment]
I would like to see a higher level OR search operator. I want to create a list of issues to further process, like: `no:assignee OR no:priority`. This would include cases like `assignee:@me OR no:assignee`, but further cases like mine as well.

**Comment by michael-hawker** [comment]
Discussed also in #16456 #16044 #15655
